extension/json/include/json_common.hpp

```diff
@@ -152,6 +172,10 @@ struct JSONCommon {
 	static inline DocPointer<yyjson_doc> ReadDocumentUnsafe(const string_t &input) {
 		return DocPointer<yyjson_doc>(yyjson_read(input.GetDataUnsafe(), input.GetSize(), READ_FLAG));
```

Comment:
Whilst we're doing this work, it would be great if we could expose the error from yyjson when it falls to parse? Makes debugging much easier